src/js/sentry-cordova.ts

```diff
@@ -39,5 +44,8 @@ export { SDK_NAME, SDK_VERSION } from './version';
 import * as Integrations from './integrations';
 export { Integrations };
 export { BrowserIntegrations };
```

Comment:
In Browser JS all integration are exported now as functions directly. `Sentry.integrationA()`

Let's add it here too, since replay, tracing and profiling is exported like this already.